perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -30,6 +53,10 @@ class GCPManagedRelationalDb(managed_relational_db.BaseManagedRelationalDb):
 
   CLOUD = providers.GCP
   SERVICE_NAME = 'managed_relational_db'
+  # These are the constants that should be specified in GCP's cloud SQL command.
+  DEFAULT_BACKUP_START_TIME = '07:00'
+  GCP_MY_SQL_VERSION = 'MYSQL_5_6'
```

Comment:
Also, don't forget about this:

```
    # Set defaults that were not able to be set in                                                                                                                                                                                      
    # GetOptionDecoderConstructions()                                                                                                                                                                                                   
    if not self.database_version:                                                                                                                                                                                                       
      managed_db_class = managed_relational_db.GetManagedRelationalDbClass(                                                                                                                                                             
          self.cloud)                                                                                                                                                                                                                   
      self.database_version = managed_db_class.GetLatestDatabaseVersion(                                                                                                                                                                
          self.database)    
```

You need to implement `GetLatestDatabaseVersion` as the spec calls that method in the case that no database version is specified. See how I do it for AWS. It will need to support Postgres and MySQL. Feel free to add a TODO and do it later if you wish.